authentik/stages/email/templates/email/account_confirmation.txt

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}{% translate "Welcome!" %}
+
+{% translate "We're excited to have you get started. First, you need to confirm your account. Just open the link below." %}
+
+{{ url }}
+
+---
+
+Powered by goauthentik.io.
```

Comment:
```suggestion
-- 
Powered by goauthentik.io.
```
It would be nice to allow a configurable message to be displayed over "Powered by goauthentik.io", especially for plaintext emails.

Reply:
Yeah the issue is when sending emails we don't have the context of a request (not always anyways) so it's not always possible to use the ~~Tenant~~ Brand for this

Reply:
Fair enough. I also saw that we don't permit it for HTML so I guess it's fine